tests/unit/schema/wrappers/test_method.py

```diff
@@ -51,10 +51,22 @@ def test_method_field_headers_none():
     assert isinstance(method.field_headers, collections.Sequence)
 
 
-def test_service_field_headers_present():
+def test_method_field_headers_present():
     http_rule = http_pb2.HttpRule(get='/v1/{parent=projects/*}/topics')
-    service = make_method('DoSomething', http_rule=http_rule)
-    assert service.field_headers == ('parent',)
+    method = make_method('DoSomething', http_rule=http_rule)
+    assert method.field_headers == ('parent',)
+
+
+def test_method_idempotent_yes():
+    http_rule = http_pb2.HttpRule(get='/v1/{parent=projects/*}/topics')
+    method = make_method('DoSomething', http_rule=http_rule)
+    assert method.idempotent is True
+
+
+def test_method_idempotent_no():
```

Comment:
Can we have a test of `assert method.idempotent is False` for a method with no `HttpRule`?

Reply:
Good idea. Done.